### PR TITLE
[Build] Allow to disable the build with the xamarin parts.

### PR DESCRIPTION
**Pull request by mandel-macaque**
Add a new option, fixed some alignment issues (an editor used tabs rather than spaces).

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -97,15 +98,19 @@ while test "x$1" != x; do
...
-		# Private option
-		echo "ENABLE_BITCODE_ON_IOS=1" >> $CONFIGURED_FILE
-		shift
-		;;
-	--disable-packaged-llvm)
-		echo "DISABLE_DOWNLOAD_LLVM=1" >> $CONFIGURED_FILE
-		shift
-		;;
+    --disable-xamarin)
+    echo "ENABLE_XAMARIN=0" >> $CONFIGURED_FILE
```
Comment (spouliot) [inline configure:102]: sure it should not be `ENABLE_XAMARIN=` (no `0`) ?

Comment (mandel-macaque) [reply configure:102]: You are right.

Comment (mandel-macaque) [inline configure:102]:
```suggestion
    echo "ENABLE_XAMARIN=" >> $CONFIGURED_FILE
```

**Review by spouliot — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Failures are are unrelated with the changes.

File: configure [hunk 2]
```diff
@@ -97,15 +98,19 @@ while test "x$1" != x; do
...
-		# Private option
-		echo "ENABLE_BITCODE_ON_IOS=1" >> $CONFIGURED_FILE
-		shift
-		;;
-	--disable-packaged-llvm)
-		echo "DISABLE_DOWNLOAD_LLVM=1" >> $CONFIGURED_FILE
-		shift
-		;;
+    --disable-xamarin)
+    echo "ENABLE_XAMARIN=" >> $CONFIGURED_FILE
```
Comment (rolfbjarne) [inline configure:102]: @mandel-macaque the indentation is wrong here.

**Pull request merged**